kubesae/info.py

```diff
@@ -1,24 +1,45 @@
 import invoke
 import yaml
 
-@invoke.task(default=True)
-def get_ansible_vars(c, var=None):
-    """Inspect ansible variables
+@invoke.task("print_ansible_vars")
+def print_ansible_vars(c, var=None, yaml=None, pty=True, hide=False):
```

Comment:
BTW, I just now noticed that `inv staging pod.fetch-namespace-var DATABASE_URL` does almost the exact same thing as this function. It currently only gets a single variable, but if we allowed it to take no arguments, then `printenv` will print out the entire environment.

Reply:
I think the need here that motivated me, is that we sometimes need to access encrypted variables that are not in the environment of a running pod. I'm thinking specifically of the admin database password. We can get it from lastpass, but if I don't have to dig into lastpass, I am happier.

Reply:
ahh gotcha, yes there are some variables that are in our ansible config, but NOT in the k8s containers, and you also might want to be able to print variables without having a running k8s setup yet. Carry on!